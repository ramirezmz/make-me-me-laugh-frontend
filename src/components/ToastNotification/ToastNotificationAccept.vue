<template>
  <div class="container_message">
    <p>{{ props.message }}</p>
  </div>
</template>
<script setup lang="ts">
const props = defineProps({
  message: String,
});
</script>
<style scoped>
.container_message {
  background-color: rgb(102, 193, 101);
  border-radius: 10px;
  color: #fff;
  padding: 1rem;
}
</style>
