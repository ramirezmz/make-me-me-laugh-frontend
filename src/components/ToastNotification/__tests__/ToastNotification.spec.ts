import { describe, it, expect } from "vitest";
import { mount } from "@vue/test-utils";
import ToastNotification from "../ToastNotification.vue";
import { nextTick } from "vue";

const localMount = async (propsData: any) => {
  const wrapper = mount(ToastNotification, {
    propsData,
  });
  nextTick();
  return wrapper;
};

describe("Toast Notification", () => {
  it("renders component", async () => {
    const wrapper = await localMount({
      message: "Testando ....",
    });
    expect(ToastNotification).toBeTruthy();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
