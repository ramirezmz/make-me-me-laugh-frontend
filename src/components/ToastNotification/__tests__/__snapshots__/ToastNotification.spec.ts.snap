// Vitest Snapshot v1

exports[`Toast Notification > renders component 1`] = `
"<div class=\\"container_message\\" data-v-906bc997=\\"\\">
  <p data-v-906bc997=\\"\\">Testando ....</p>
</div>"
`;
